src/services/3box/index.ts

```diff
@@ -32,12 +34,18 @@ export const get3BoxProfile = async (wallet: Address) => {
     return user
   } catch (err) {
     console.log(err)
-    return {}
+    return { image }
   }
 }
 
-export const getUserProfile = async (wallet: string) => {
+const cache3box: {
```

Comment:
ooh, i wonder if we could put these to localstorage or indexdb too... since they are just so slow lol

Reply:
yeah, would def help. I'll just add a comment about it for now.